src/software/simulated_tests/simulated_er_force_sim_play_test_fixture.cpp

```diff
@@ -85,6 +85,7 @@ void SimulatedErForceSimPlayTestFixture::updatePrimitives(
 
     auto primitive_set_msg = ai.getPrimitives(world_with_updated_game_state);
     LOG(VISUALIZE) << ai.getPlayInfo();
+    LOG(VISUALIZE) << *primitive_set_msg;
```

Comment:
hm, what is this for?

Reply:
The PrimitiveSet (path + static obstacles) were never being visualized during C++ tests, so had to add this.